### ISSUE TITLE
Renaming

### ISSUE THREAD

**Issue by Nero1024x** [upscayl/upscayl#966]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [x] I have searched for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).

### Description

Can we have option not to rename files, but leave output names as is? And I have no knowledge of coding. Thank you

**Comment by abhishek-gaonkar** [comment]
Hi @Nero1024x 
I am slightly confused by your question. Did you want the output file to have the same name as the input file?
If yes, then that's not possible since you cannot have two files with the same name inside a folder. 
If you meant something else, please do elaborate

**Comment by Nero1024x** [comment]
@abhishek-gaonkar Yes. Option to keep the same name as input files, but when output is different folder than input folder

**Comment by abhishek-gaonkar** [comment]
Understood.. @NayamAmarshe shall I take this?

**Comment by NayamAmarshe** [comment]
> Understood.. @NayamAmarshe shall I take this?

We're already doing this for batch upscayl. The file names will have the same names.
